src/pages/workspaces/workspace/analysis/AppLauncher.js

```diff
@@ -174,7 +174,7 @@ const ApplicationLauncher = _.flow(
         await Ajax()
           .Runtimes
           .fileSyncing(googleProject, runtime.runtimeName)
-          .setStorageLinks(localBaseDirectory, cloudStorageDirectory, getPatternFromTool(getToolFromRuntime(runtime))) :
+          .setStorageLinks(localBaseDirectory, ``, cloudStorageDirectory, getPatternFromTool(getToolFromRuntime(runtime))) :
```

Comment:
@Qi77Qi Sorry for the late comment: Backticks (``) in JS are sort of like `s""` in Scala, meant to be used only when some variable substitution happens. When that's not the case, we use plain quotes ('').

Reply:
ah, good to know for future reference... I was following https://github.com/DataBiosphere/terra-ui/blob/0add265b8ac0c96f4239e8864e4b86719910192a/src/pages/workspaces/workspace/analysis/AppLauncher.js#L167 here....

Reply:
Our ESLint configuration explicitly allows this: https://github.com/DataBiosphere/terra-ui/blob/e6fcd9639d1abff9c23c3fe35cf3f803b3c79c95/.eslintrc.js#L57

https://eslint.org/docs/latest/rules/quotes

Running ESLint without that configuration finds 117 places where we use template strings without substitution.

Reply:
I was going to make a PR to change the linter behaviour but now that I know it's 117 places, I'm not so sure 😛

Kidding…I'll still try to get to it soon 😉 Thank you @nawatts for keeping me honest that my comment wasn't fully accurate in that we hadn't been consistent about applying the convention.

Reply:
> Also, this sort of thing is easy to change automatically with [eslint --fix](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix)

Oh right -- doh :)

Reply:
PR: https://github.com/DataBiosphere/terra-ui/pull/3437